### PR TITLE
fix vae entry

### PR DESCRIPTION
**Pull request by jburel**
Fix entry for VAE

### CODE REVIEW

File: ansible/group_vars/omero-hosts.yml [hunk 1]
```diff
@@ -517,8 +517,8 @@ omero_web_apps_config_set:
           href: "https://idr.openmicroscopy.org/about/download.html"
         - text: "Image Tools Resource (ITR)"
           href: "https://idr.openmicroscopy.org/about/itr.html"
-        - text: "Virtual Analysis Environment (VAE)"
-          href: "https://idr.openmicroscopy.org/jupyter"
+        - text: "Analysis Environments"
+          href: "https://idr.openmicroscopy.org/analysis-environments.html"
```
Comment (sbesson) [inline ansible/group_vars/omero-hosts.yml:521]:
```suggestion
          href: "https://idr.openmicroscopy.org/about/analysis-environments.html"
```

?

**Review by sbesson — CHANGES_REQUESTED** [review]

**Comment by jburel** [comment]
well spotted

**Comment by sbesson** [comment]
This should now be updated with https://github.com/IDR/idr.openmicroscopy.org/releases/tag/2021.09.20

**Comment by jburel** [comment]
@sbesson update pushed

**Review by sbesson — APPROVED** [review]
Successfully deployed on `prod101`

File: ansible/group_vars/proxy-hosts.yml [hunk 2]
```diff
@@ -261,7 +261,7 @@ nginx_proxy_direct_locations:
 - location: "^~ /mineotaur"
   redirect302: /about/mineotaur.html
 - location: "^~ /jupyter"
-  redirect302: https://idr-analysis.openmicroscopy.org
+  redirect302: /about/linked-resources.html
```
Comment (sbesson) [inline ansible/group_vars/proxy-hosts.yml:264]: Realized a bit too late. Should this have been

```diff
-  redirect302: https://idr-analysis.openmicroscopy.org
+  redirect302: /about/analysis-environments.html
```

instead?

Comment (jburel) [reply ansible/group_vars/proxy-hosts.yml:264]: yes

**Pull request merged**